src/construction.cpp

```diff
@@ -1556,8 +1556,9 @@ int construction::adjusted_time() const
 }
 
 std::string construction::get_time_string() const
-{
-    const int turns = adjusted_time() / 100;
+{    
+    const float season_mult = calendar::turn.season_length()/91.0;
+    const int turns = (int)(season_mult * adjusted_time() / 100);
```

Comment:
It would probably be easier if you just do the adjustment when the moves get calculated, e.g. in `construction::adjusted_time`, instead of figuring out where it's used.

Similar for recipes: make the `time` member private and add a function that returns the actual time (including skill etc.), which can also be used for display.

Also, the adjustment of the moves in the activity handling is surprising, the adjustment should be done once in the code that sets up the activity. Those places should be easy to find by searching for `assign_activity`.